packages/base/src/StaticAreaItem.js

```diff
@@ -0,0 +1,87 @@
+import { getStaticAreaInstance, removeStaticArea } from "./StaticArea.js";
+
+let staticAreaIndex = 1;
+
+/**
+ * @class
+ * @author SAP SE
+ * @private
+ * Defines and takes care of ui5-static-are-item items
+ */
+class StaticAreaItem {
+	constructor(_ui5ElementContext) {
+		this.ui5ElementContext = _ui5ElementContext;
+	}
+
+	/**
+	 * @protected
+	 */
+	_updateFragment() {
+		const renderResult = this.ui5ElementContext.constructor.staticAreaTemplate(this.ui5ElementContext),
+			stylesToAdd = this.ui5ElementContext.constructor.staticAreaStyles || false;
+		let domNodeToRenderInCurrentComponent;
+
+		if (!this.currentStaticAreaPosition) {
+			// Initial rendering of fragment
+			this.currentStaticAreaPosition = this.constructor._staticAreaPosition;
+
+			domNodeToRenderInCurrentComponent = document.createElement("ui5-static-area-item");
```

Comment:
if we store this node in the instance, the whole currentStaticAreaPosition can be removed